php/class-currency.php

```diff
@@ -0,0 +1,91 @@
+<?php namespace Yoast\YoastCom\VisitorCurrency;
+
+/**
+ * Class Currency
+ * @package Yoast\YoastCom\VisitorCurrency
+ */
+class Currency {
+	private $code;
+	private $label;
+	/**
+	 * @var bool
+	 */
+	private $enabled;
+	/**
+	 * @var bool
+	 */
+	private $default;
+
+	/**
+	 * Currency constructor.
+	 *
+	 * @param string $code
+	 * @param string $label
+	 * @param bool $enabled
+	 * @param bool $default
+	 */
+	public function __construct($code, $label = '', $enabled = true, $default = false)
+	{
+		$this->code = $code;
+		$this->label = $label;
+		$this->enabled = $enabled;
+		$this->default = $default;
+	}
+
+	/**
+	 * Gets the code.
+	 *
+	 * @return string The currency code.
+	 */
+	public function get_code() {
+		return $this->code;
+	}
+
+	/**
+	 * Gets the label.
+	 *
+	 * @return string The label.
+	 */
+	public function get_label() {
+		return $this->label;
+	}
+
+	/**
+	 * Determines whether or not the currency is the default currency.
+	 *
+	 * @return bool Whether or not the currency is the default currency.
+	 */
+	public function is_default() {
```

Comment:
By simply returning the property, you might run into an issue with type juggling. This way ensures that we're actually getting a boolean back. Once we move over to strict typing (if ever) this could certainly be changed.